rand_distr/src/normal.rs

```diff
@@ -122,14 +122,18 @@ where F: Float, StandardNormal: Distribution<F>
 /// Error type returned from `Normal::new` and `LogNormal::new`.
 #[derive(Clone, Copy, Debug, PartialEq, Eq)]
 pub enum Error {
-    /// `std_dev < 0` or `nan`.
-    StdDevTooSmall,
+    /// The mean value is too small (log-normal samples must be positive)
+    MeanTooSmall,
+    /// The standard deviation or other dispersion parameter is negative,
+    /// infinite, or not-a-number.
+    BadVariance,
 }
 
 impl fmt::Display for Error {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         f.write_str(match self {
-            Error::StdDevTooSmall => "std_dev < 0 or is NaN in normal distribution",
+            Error::MeanTooSmall => "mean < 0 or NaN in log-normal distribution",
+            Error::BadVariance => "variation parameter is negative or non-finite in (log)normal distribution",
```

Comment:
Don't we allow negative standard deviations now?

Reply:
I think this is the only remaining concern, otherwise this seems ready to merge.

Reply:
Done